galasa-ui/src/tests/components/UserRoleSelection.test.tsx

```diff
@@ -60,20 +60,21 @@ const dummyProfile: UserData = {
         id: '1',
         name: 'tester',
         description: 'Test developer and runner',
+        assignable: true
       },
     },
   },
 };
 
 const dummyRoles: RBACRole[] = [
   {
-    metadata: { id: '1', name: 'tester', description: 'Test developer and runner' },
+    metadata: { id: '1', name: 'tester', description: 'Test developer and runner', assignable:true },
   },
   {
-    metadata: { id: '2', name: 'admin', description: 'Administrator' },
+    metadata: { id: '2', name: 'admin', description: 'Administrator', assignable:true },
   },
   {
-    metadata: { id: '0', name: 'deactivated', description: 'User has no access' },
+    metadata: { id: '0', name: 'deactivated', description: 'User has no access', assignable:true },
```

Comment:
Perhaps we could add a mock role that isn't assignable and test that it doesn't appear in the dropdown?